platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Utilities.java

```diff
@@ -48,13 +48,11 @@ private Utilities() {}
 
     /**
      * Convert a string to a boolean.
+     * <p>
+     * A false is defined to be any string that, after trimming leading/trailing whitespace and conversion to lowercase,
```

Comment:
Minor: it would be nice to avoid massive code reformatting in PRs, unless it's enforced by spotless or similar mandatory build logic. This is because massive reformatting makes it more difficult to navigate the git history in the future. Also, it makes it difficult to review such code changes because it unnecessarily enlarges them. If the massive, voluntary (not enforced by a tool) reformatting must be released for whatever reason, this can be done in a separate, dedicated PR.

Reply:
It is unfortunate that the formatting by IntelliJ is slightly different.  I unconsciously auto format using keyboard shortcuts.   At this point, if I switch it back and extend the commit history, you'll have the same thin in reverse.

Reply:
Not really. The `develop` will only receive a squashed commit. And I only ever care about the commit history in the `develop` branch. So undoing the reformatting in this PR while it's still open wouldn't be a problem.

Reply:
Also, IntelliJ formatting _should not_ rewrite the javadoc this way if you have the "official" style settings installed.
Note, github won't allow .xml uploads (even if the content is xml; it requires a `.txt` extension).  This file must be saved with `.xml` extension to import into IntelliJ.
[Hashgraph_Style.xml](https://github.com/user-attachments/files/17835249/Hashgraph_Style.xml.txt)

Reply:
I have imported the new style, thank you.

Reply:
Using the new style, the formatter for intellij still puts the javadoc params on the same line.   I'll undo the style changes to the older javadoc